### DISCUSSION TITLE
Share your themes!

### DISCUSSION THREAD

**Discussion by Neuropsihopuppu** [mathoudebine/turing-smart-screen-python#58] [Themes]
Is there a specific place i could get more themes or is anybody willing to share their creations?

Sorry for my bad english!

Thanks in advance!

**Comment by mathoudebine** [comment]
Hi! I created the "Themes" category on the discussion space on Github
https://github.com/mathoudebine/turing-smart-screen-python/discussions/categories/themes
Feel free to share the themes you've made here!
I also try to add more themes when i have some time

**PS: this conversation is now locked, to share your theme please create a discussion in the "Themes" space using this link: https://github.com/mathoudebine/turing-smart-screen-python/discussions/new?category=themes
Thanks!**

**Comment by superlulu55** [comment]
I made an Ironman theme. It works very fine and I love it for my Raspberry Pi !
![ironman](https://user-images.githubusercontent.com/79225820/203648707-6f043068-5c9d-454d-9c0a-3d9ea02ece77.jpg)
![background](https://user-images.githubusercontent.com/79225820/203648113-796d4698-117d-4f6a-8d16-f3a75e650b7a.png)
[ironman.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/10079243/ironman.zip)

**Reply by mathoudebine** [reply]
Thanks for sharing this theme with the community!

**Comment by Kunoz07** [comment]
![Portrait 5](https://user-images.githubusercontent.com/121983479/210663324-994c987a-6489-4482-8883-db74ef566014.jpg)
[Turing 3.5 - Portrait 5.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/10347957/Turing.3.5.-.Portrait.5.zip)

**Reply by mathoudebine** [reply]
Thanks for sharing, very nice theme!

**Comment by drxxali** [comment]
hi
ihave qs
my screen is 3.5 inch  from where ican get new themes
and how i add them 
can any one give us detailed information pplease and thnx

**Comment by Pepper-the-kobold** [comment]
Only a relatively minor edit, but I like the higher contrast and the red bars match my PC lighting. It's not pixel perfect but the display softens the edges and it looks fine in person.
![image](https://user-images.githubusercontent.com/125101447/218227131-55277ea9-8710-42f0-afb4-2e2acf377ab6.png)
![background](https://user-images.githubusercontent.com/125101447/218227137-79aec3ac-14a1-486d-afe2-c432bb519b20.png)
Background is 50, 50, 50 (323232)
Bars are 255, 0, 0

**Reply by mathoudebine** [reply]
Looks very nice in this "dark" version! Thanks for sharing!
